nav/src/main/java/io/stanwood/framework/arch/nav/NavigationViewSyncer.kt

```diff
@@ -38,7 +38,9 @@ class NavigationViewSyncer(
 
     val destinationChangedListener: (NavController, NavDestination, Bundle?) -> Unit = { _, destination, _ ->
         bottomNav?.also {
-            it.selectedItemId = destination.id
+            it.menu.findItem(destination.id)?.apply {
```

Comment:
@aclassen Isn't this a straight revert from your last PR? https://github.com/stanwood/framework-arch-android/pull/26

Didn't it work out in the end?

Reply:
stack overflow

Reply:
@aclassen Do you have a link to the thread? Would be interested in the discussion.

Reply:
it results in a stack overflow :D @ubuntudroid